README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+<h2> Michael Lynch </h2>
+<h3>Name</h3>
+I like my name, but it is absurdly common. Not only am I the fourth Michael Lynch in a row (although not Michael the Fourth, since my middle name is different), I am one of nearly *5 million people* in the United States named Michael. That's more people than the **entire state** of Alabama.
+
+<h3>Geography</h3>
+I was born in Boston, Massachusetts, and lived there until I was four, when my family moved to Georgia so my dad could work for the Centers for Disease Control. I grew up in Dunwoody, Georgia, a suburb north of Atlanta. There's not much there other than some trees and hills, but Ryan Seacrest went to my high school. So that's fun. 
+
+![These two towers are called the King and Queen](C:\Users\mtlyn\Pictures\Book Cover\Downtown dunwoody.jpg "Downtown Dunwoody")
```

Comment:
I think you should use the path of Github here, instead of the path of your drive.

Reply:
Yes. You are using an absolute file path that makes github failing at locating your image. Try using a relative file path and uploading the image next time.